schemas/ns10_cache_entry.fbs

```diff
@@ -0,0 +1,13 @@
+
+file_identifier "ns10";
+
+
+/// pylint: skip-file
+table CacheEntryFB {
+        time:double;
+	ttl:double;
+	expired:bool = false;
+	value:string;
```

Comment:
Are all NICOS cache values just strings?

Reply:
Yes, the internal structure assumes all values to be strings. The cache daemon in NICOS needs all values to be strings. I know that this might not be optimal, but changing this would need a lot of changes in NICOS and it's processes and I would rather skip that.